subsys/bluetooth/shell/has.c

```diff
@@ -75,6 +75,43 @@ static int cmd_preset_unreg(const struct shell *sh, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_has_register(const struct shell *sh, size_t argc, char **argv)
+{
+	int err;
+	struct bt_has_register_param param = {
+		.type = BT_HAS_HEARING_AID_TYPE_MONAURAL,
+		.preset_sync_support = false,
+		.independent_presets = false
+	};
+
+	for (size_t argn = 1; argn < argc; argn++) {
+		const char *arg = argv[argn];
+
+		if (strcmp(arg, "binaural") == 0) {
+			param.type = BT_HAS_HEARING_AID_TYPE_BINAURAL;
+		} else if (strcmp(arg, "monaural") == 0) {
+			param.type = BT_HAS_HEARING_AID_TYPE_MONAURAL;
+		} else if (strcmp(arg, "banded") == 0) {
+			param.type = BT_HAS_HEARING_AID_TYPE_BANDED;
```

Comment:
Since these values are basically a choice, rather than 3 individual parameters, I would suggest to keep the hearing aid type as a mandatory argument to `has register` and leave the `sync` and `independent` as optional parameters. 

Basically changing

```
"[binaural | monaural(default) | banded] [sync] [independent]",
cmd_has_register, 1, 3),
```
to 
```
"<binaural | monaural | banded> [sync] [independent]",
cmd_has_register, 2, 2),
```

Does that make sense?

Reply:
I like to have a default for convenience - something that works on the minimal setup with just one device. Maybe @fredrikdanebjer or @MariuszSkamra has an opinion on this?